{"uuid": "0193a55a-49e9-455e-9553-189490caf302", "children": ["c9b18390-fedb-41c9-8afd-1a83623ac46a"], "befores": [{"name": "_Class__pytest_setup_method", "status": "passed", "start": 1553479365301, "stop": 1553479374931}], "afters": [{"name": "_Class__pytest_setup_method::0", "status": "broken", "statusDetails": {"message": "AttributeError: 'TestCustomerManageAdd' object has no attribute 'driver'\n", "trace": "  File \"D:\\softwore\\python372\\lib\\site-packages\\allure_commons\\_allure.py\", line 169, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"D:\\softwore\\python372\\lib\\site-packages\\_pytest\\fixtures.py\", line 790, in _teardown_yield_fixture\n    next(it)\n  File \"D:\\softwore\\python372\\lib\\site-packages\\_pytest\\python.py\", line 777, in xunit_setup_method_fixture\n    _call_with_optional_argument(func, method)\n  File \"D:\\softwore\\python372\\lib\\site-packages\\_pytest\\python.py\", line 691, in _call_with_optional_argument\n    func()\n  File \"D:\\my-git-project\\91lng\\selenium_91lng_auto_test\\test_case\\test_bpm\\test_customer_manage.py\", line 49, in teardown_method\n    self.driver.close()\n"}, "start": 1553479407243, "stop": 1553479407243}], "start": 1553479365301, "stop": 1553479407243}